src/main/java/org/spongepowered/common/command/CommandPermissions.java

```diff
@@ -53,15 +53,20 @@ public static boolean testPermission(CommandSource source, String commandName) {
         if (commandName.equals(CommandPermissions.COMMAND_BLOCK_COMMAND)) {
             return source.hasPermission(CommandPermissions.COMMAND_BLOCK_PERMISSION);
         }
-        Optional<? extends CommandMapping> mapping = SpongeImpl.getGame().getCommandManager().get(commandName);
-        if (mapping.isPresent()) {
-            CommandCallable callable = mapping.get().getCallable();
-            if (callable instanceof MinecraftCommandWrapper) {
-                return source.hasPermission(((MinecraftCommandWrapper) callable).getCommandPermission());
-            } else {
-                return callable.testPermission(source);
```

Comment:
This else seemed to be unused / useless as this method was only ever called on native commands.